server/src/capture.rs

```diff
@@ -0,0 +1,198 @@
+use log::{error, info};
+use serde::Deserialize;
+use simplelog::*;
```

Comment:
I'm already using log4rs, would you use that instead? See `webserver.rs configure_logger` and `server/log4rs.yml`.